### PR TITLE
Pei-Ling 20

### PR DESCRIPTION
**Pull request by wuberde**

### CODE REVIEW

File: 20-PB-program-multiplechoice/Pei-Ling.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    case `orange`:
+        console.log(`Oranges are 1.99 €/kg.`);
+        break;
+    case `strawberry`:
+        console.log(`Strawberries are 1.49 €/500g.`);
+        break;
+    case `apple`:
+        console.log(`Apple are 1.49 €/kg.`);
+        break;
+}
```
Comment (martl) [inline 20-PB-program-multiplechoice/Pei-Ling.js:61]: don't forget a default error case

File: 20-PB-program-multiplechoice/Pei-Ling.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+// #### 4. Percentage Complete.
+let percentageComplete = 50;
+switch (percentageComplete === 50) {
```
Comment (martl) [inline 20-PB-program-multiplechoice/Pei-Ling.js:65]: you need to say switch(true)

File: 20-PB-program-multiplechoice/Pei-Ling.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        console.log(`Slowly getting there.`);
+        break;
+    case 51 <= percentageComplete <= 80:
+        console.log(`You can do it!`);
+        break;
+    case 81 <= percentageComplete <= 99:
+        console.log(`This is the last push!`);
+    case percentageComplete === 100:
+        console.log(`You're there. Well done!`);
+}
```
Comment (martl) [inline 20-PB-program-multiplechoice/Pei-Ling.js:79]: don't forget a default error case

File: 20-PB-program-multiplechoice/Pei-Ling.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        console.log(`Still a long way to go.`);
+        break;
+    case 30 <= percentageComplete <= 50:
+        console.log(`Slowly getting there.`);
+        break;
+    case 51 <= percentageComplete <= 80:
+        console.log(`You can do it!`);
+        break;
+    case 81 <= percentageComplete <= 99:
+        console.log(`This is the last push!`);
```
Comment (martl) [inline 20-PB-program-multiplechoice/Pei-Ling.js:76]: dont forget break

File: 20-PB-program-multiplechoice/Pei-Ling.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    case percentageComplete === 100:
+        console.log(`You're there. Well done!`);
```
Comment (martl) [inline 20-PB-program-multiplechoice/Pei-Ling.js:78]: dont forget break

Comment (wuberde) [reply 20-PB-program-multiplechoice/Pei-Ling.js:81]: fixed

File: 20-PB-program-multiplechoice/Pei-Ling.js (hunk 4 above)
Comment (wuberde) [reply 20-PB-program-multiplechoice/Pei-Ling.js:78]: fixed

File: 20-PB-program-multiplechoice/Pei-Ling.js (hunk 2 above)
Comment (wuberde) [reply 20-PB-program-multiplechoice/Pei-Ling.js]: fixed

File: 20-PB-program-multiplechoice/Pei-Ling.js (hunk 1 above)
Comment (wuberde) [reply 20-PB-program-multiplechoice/Pei-Ling.js:63]: fixed

**Pull request closed without merging**